deepchecks/tabular/checks/integrity/outlier_sample_detection.py

```diff
@@ -94,10 +95,13 @@ def run_logic(self, context: Context, dataset_type: str = 'train') -> CheckResul
         # Calculate distances matrix and retrieve nearest neighbors based on distance matrix.
         df_cols_for_gower = df[dataset.cat_features + dataset.numerical_features]
         is_categorical_arr = np.array(df_cols_for_gower.columns.map(lambda x: x in dataset.cat_features), dtype=bool)
+        df_cols_for_gower.iloc[:, is_categorical_arr] = OrdinalEncoder(handle_missing=np.nan
```

Comment:
Why the conversation for ordinal, is it for performance or for null handling?

Reply:
please use run_available_kwargs for OrdinalEncoder